modules/storages/spec/common/storages/peripherals/nextcloud_connection_validator_spec.rb

```diff
@@ -255,11 +255,11 @@
     end
 
     it "returns a validation failure if storage audience isn't set" do
-      storage.update(nextcloud_audience: nil)
+      storage.update!(nextcloud_audience: nil)
 
-      expect(subject.type).to eq(:error)
-      expect(subject.error_code).to eq(:oidc_audience_missing)
-      expect(subject.description).to eq(I18n.t("storages.health.connection_validation.oidc_audience_missing"))
+      expect(subject.type).to eq(:none)
+      expect(subject.error_code).to eq(:wrn_not_configured)
+      expect(subject.description).to eq(I18n.t("storages.health.connection_validation.wrn_not_configured"))
```

Comment:
@mereghost FYI: I removed the separate audience check from the validator, because after adding a configuration check for the audience to the storage (which was necessary for a complete/incomplete label), the generic configuration check of the validator took control of the error result.

I left this spec here intact, to define the new expected behaviour.